docs/bsp.md

```diff
@@ -792,6 +884,18 @@ trait TestReport {
 
   /** The total number of failed tests. */
   def failed: Int
+  
+  /** The total number of ignored tests. */
+  def ignored: Int
+
+  /** The total number of cancelled tests. */
+  def cancelled: Int
+
+  /** The total number of skipped tests. */
```

Comment:
What's the difference between ignored and skipped?

Reply:
I think ignored are tests ignored by the user (with `@Ignored`), whereas skipped are tests not run by the build tool. I got these fields from sbt's test interface design.